codebase/config/sync/migrate_plus.migration.idc_ingest_new_items.yml

```diff
@@ -299,7 +299,10 @@ process:
             bundle: language
             bundle_key: vid
             value_key: field_language_code
-  field_date_available: date_available
+  field_date_available:
```

Comment:
Big Picture: this field is now optional.

Reply:
oh, hmm, no that should not be the case.  It's still a required field. The skip on empty simple leaves it unset (versus setting to a NULL).  The validation code run after this will check for it being set since it's required. 

To my knowledge, setting a field like this does not make it required:  `field_date_available: date_available`
It will simply give it a null value (when the cell is empty) versus leaving it unset.

Reply:
I think in practice we should use skip_on_empty in all places.

Reply:
So `skip_on_empty` will force the value to be unset, thus failing validation, if a value isn't provided?  Previously, without `skip_on_empty`, a `NULL` would implicitly be added, and therefore allowing the record to validate in cases when no date was provided?

Reply:
w/ or w/o skip_on_empty here, this field is explicitly required by drupal.  The skip_on_empty just stops processing in the case the field is empty and leaves it unset.  In either case, the field should still be invalid.

Reply:
one thing we can do with skip_on_empty is provide a message to be displayed in the event of the field being empty. That's pretty handy.